service/docs/source/geopm_agent_cpu_activity.7.rst

```diff
@@ -126,7 +126,7 @@ Without this, the uncore bandwidth characterization analysis scripts will not
 be able to accurately determine the maximum memory bandwidth at each uncore
 frequency.
 
-Additionally the ``test.sbatch`` should be modified to include the following
+Additionally the ``test.sbatch`` or ``test.pbs`` should be modified to include the following
 experiment options, where the text within angle brackets (``<>``) needs to be
 replaced with relevant system (or administrator chosen) values::
 
```

Comment:
Some notes for other details to update:
- The text below is still mainly based on Slurm, so it could be updated to also take PBS into account. Namely, there are references to `test.sbatch` and Slurm launch commands. Maybe we could use that nice box with a tab for each option (Slurm, PBS), as we do in our Getting Started guide (for Python, C, C++ code examples).
- In the note about extending the run time, add the line that needs to be updated (e.g. `#PBS -l walltime` for PBS).